plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -714,6 +714,22 @@ enterComputeCek = computeCek
             Nothing -> throwingDischarged _MachineError (MissingCaseBranch i) e
         _ -> throwingDischarged _MachineError NonConstrScrutinized e
 
+    -- | Push arguments onto the stack. The first argument will be the most recent entry.
+    pushArgs
+        :: Spine (CekValue uni fun ann)
+        -> Context uni fun ann
+        -> Context uni fun ann
+    pushArgs args ctx = foldr FrameAwaitFunValue ctx args
```

Comment:
This is similar to `transferArgStack`, except that the latter is a left fold, and operates on `ArgStack` instead of `Spine`. Perhaps put them close to each other, and give them similar names.

Reply:
Great point, thanks. Done.